### PR TITLE
added university course listing

### PR DESCRIPTION
**Pull request by bwalsh5**

### CODE REVIEW

File: content/learn.md [hunk 1]
```diff
@@ -81,6 +81,8 @@ We highly recommend pairing R4DS with the [RStudio cheatsheets](https://www.rstu
...
+
+* [Applied Media Analytics](https://www.elon.edu/u/academics/communications/academics/media-analytics-major/) Elon University; Brian Walsh. An Undergraduate course for Media Analytics majors, students are introduced to R programming via the tidyverse and learn ggplot, dplyr, & lubridate, as well as basic sentiment analysis, Twitter analysis, and Google Analytics insights.
```
Comment (batpigandme) [inline content/learn.md]: I know it's inconsistent across the listings, but I'd suggest maybe breaking this into sentences. "This is an undergraduate course..." And then have a full stop before "Students are introduced to R programming via the tidyverse." Then, maybe rework that last sentence to start with: "Students will learn ggplot**2**, dplyr...".

Let me know if you have any questions.

Comment (bwalsh5) [reply content/learn.md]: Thanks for the feedback! Will do.

**Comment by batpigandme** [comment]
Looks good. @hadley, any other steps before merging?

**Comment by hadley** [comment]
@batpigandme nope, and you have the con for merging

**Pull request merged**